<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>821</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(42, 42, 42);  border: none;
    margin: 0px;
    padding: 0px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="Data_Grid">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>1131</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    background-color: #3C3D37;
    border-radius: 10px;
    border: 1px solid #5A5A5A;
    color: whitesmoke;
    gridline-color: #5A5A5A;
}

QHeaderView::section {
    background-color: #5A5A5A;
    color: whitesmoke;
    padding: 4px;
    border: 1px solid #3C3D37;
}

QTableView::item {
    border: none;
    padding: 4px;
}

QTableView::item:selected {
    background-color: #6A6A6A;
    color: white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>27</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:whitesmoke;

</string>
    </property>
    <property name="text">
     <string>GitHub Scrapping</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txt_algorithm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0.5px solid whitesmoke;
border-radius:5px;
color:whitesmoke;
padding:0px auto;
font-size:15px;
font-weight:500;

</string>
    </property>
    <property name="currentText">
     <string>Bubble Sort</string>
    </property>
    <property name="minimumContentsLength">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Bubble Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selection Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Count Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radix Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bucket Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heap Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shell Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hybrid Merge Sort</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="txt_sorting_base">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0.5px solid whitesmoke;
border-radius:5px;
color:whitesmoke;
padding:0px auto;
font-size:15px;
font-weight:500;

</string>
    </property>
    <item>
     <property name="text">
      <string>Ascending </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:0.5px solid whitesmoke;
color: rgb(255, 255, 255);
border-radius:10px;
font-size:20px;
font-weight:500;
background-color: rgb(0, 181, 0);</string>
    </property>
    <property name="text">
     <string>Start Sorting</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_search_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>871</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Trebuchet MS&quot;;
border-radius:5px;
border:0.5px solid white;
padding:4px auto;
color:whitesmoke;
</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Trebuchet MS'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>search something....</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>732</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid whitesmoke;
    border-radius: 10px;
    text-align: center;
    color: whitesmoke;
}

QProgressBar::chunk {
    background-color: #05B8CC; /* Set the filling color here */
	background-color: rgb(0, 206, 0);
    width: 20px;
}</string>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:whitesmoke;
border-radius:10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:whitesmoke;

</string>
    </property>
    <property name="text">
     <string>Runtime :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CrossButton">
    <property name="geometry">
     <rect>
      <x>1374</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1432</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
